src/overlays/actors/ovl_En_Gamelupy/z_en_gamelupy.h

```diff
@@ -8,10 +8,14 @@ struct EnGamelupy;
 typedef void (*EnGamelupyActionFunc)(struct EnGamelupy*, PlayState*);
 
 typedef struct EnGamelupy {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x4C];
-    /* 0x0190 */ EnGamelupyActionFunc actionFunc;
-    /* 0x0194 */ char unk_194[0x10];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ ColliderCylinder collider;
+    /* 0x190 */ EnGamelupyActionFunc actionFunc;
+    /* 0x194 */ s32 rupeeIndex;
+    /* 0x198 */ s16* unk_198; //! TODO: Look at type based on EnLookNuts decomp
```

Comment:
```suggestion
    /* 0x198 */ s16* unk_198;
```
So looking at the EnLookNuts PR, this is in fact an `s16*`, so the todo is no longer needed.